docs/upgrading.md

```diff
@@ -5,6 +5,14 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.4.0` to `v0.5.0`
+
+
+One backwards incompatible changes
+## Service component
+
+- This release introduces a new format for defining multiple overrides for an agent on a service configuration. Please follow this [guide](https://docs.autonolas.network/guides/service_configuration_file/) to update your service configurations accordingly. 
```

Comment:
This link will work after the new docs release

Reply:
Turns out this was not really a good idea because this lead to the doc link check failure, so I updated the reference to the markdown file on the git repo [c533f22](https://github.com/valory-xyz/open-autonomy/pull/1616/commits/c533f22c19b29d5b46a190d1b36f2c232e58e4db)